lib/fake_artist/hostess.ex

```diff
@@ -3,7 +3,7 @@ defmodule FakeArtist.Hostess do
   require Logger
 
   @chars "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
-  @length_of_room_name 4
+  @length_of_room_name 1
```

Comment:
Practically, this limits how many "lobbies that haven't yet started the game" we can have going concurrently. I figure 26 is plenty for now, though we should bump this to 2 letters before launch (I don't think we'll ever see 26^2)